zulipterminal/ui_tools/views.py

```diff
@@ -1462,3 +1463,55 @@ def keypress(self, size: urwid_Size, key: str) -> str:
             )
             return key
         return super().keypress(size, key)
+
+
+class EmojiPickerView(PopUpView):
+    """
+    Displays Emoji Picker view for messages.
+    """
+    def __init__(self, controller: Any, title: str, emoji_names: List[str],
+                 message: Message) -> None:
+        self.width = 40
```

Comment:
In the third commit, pressing <kbd>enter</kbd> in the emoji search box causes ZT to crash. Traceback logs show that `EmojiPickerView` has no view attribute.